R/db.R

```diff
@@ -45,6 +45,7 @@ setMethod(
 # Simple class prototype to avoid messages about unknown classes from setMethod
 setClass("Oracle", where = class_cache)
 
+#' @rdname DBI-methods
```

Comment:
These were previously documented in Rd files made by S4, which is why we had those weird titles containing `~~~.